02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,114 @@
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Nunito' rel='stylesheet' type='text/css'>
+    <title>Enrique Urbina</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
 
+    <!--
+<div id="bar">
+        <br>
+        <br>
+    </div>
+    <div>-->
+
+
+    <h1 id="name">Enrique Urbina</h1>
+
+    </div>
+    <img width="100%" height="40px" src="http://wallpoper.com/images/00/37/24/62/minimalistic_00372462.jpg" alt="Sorry:(">
+    <div id="location">
+        <h3>Location</h3>
+        <p>New Braunfels, TX, 78130</p>
+    </div>
+
+    <div id="about">
+
+        <h1 id="about2">About</h1>
+
+        <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Qui beatae, quod quos sunt laboriosam? Magnam facilis, non delectus. Veritatis, enim, perferendis. Earum at quae excepturi libero laboriosam, officiis quas, exercitationem? Lorem ipsum
+            dolor sit amet, consectetur adipisicing elit. Qui beatae, quod quos sunt laboriosam? Magnam facilis, non delectus. Veritatis, enim, perferendis. Earum at quae excepturi libero laboriosam, officiis quas, exercitationem? Lorem ipsum dolor sit
+            amet, consectetur adipisicing elit. Qui beatae, quod quos sunt laboriosam? Magnam facilis, non delectus. Veritatis, enim, perferendis. Earum at quae excepturi libero laboriosam, officiis quas, exercitationem? Lorem ipsum dolor sit amet, consectetur
+            adipisicing elit. Qui beatae, quod quos sunt laboriosam? Magnam facilis, non delectus. Veritatis, enim, perferendis. Earum at quae excepturi libero laboriosam, officiis quas, exercitationem? Lorem ipsum dolor sit amet, consectetur adipisicing
+            elit. Qui beatae, quod quos sunt laboriosam? Magnam facilis, non delectus. Veritatis, enim, perferendis. Earum at quae excepturi libero laboriosam, officiis quas, exercitationem? Lorem ipsum dolor sit amet, consectetur adipisicing elit. Qui
+            beatae, quod quos sunt laboriosam? Magnam facilis, non delectus. Veritatis, enim, perferendis. Earum at quae excepturi libero laboriosam, officiis quas, exercitationem?</p>
+
+    </div>
+
+    <div id="list">
+        <h1 class="list2">Interest</h1>
+
+        <ul>
+            <a href="#">
+                <li>Coding</li>
+            </a>
+            <a href="#">
+                <li>Cars</li>
+            </a>
+            <a href="#">
+                <li>Movies</li>
+            </a>
+        </ul>
+        <hr>
+        <div>
+            <h1 class="list2">Things I Don't Like</h1>
+            <ul>
+                <a href="#">
+                    <li>Traffic</li>
+                </a>
+                <a href="#">
+                    <li>Rain</li>
+                </a>
+                <a href="#">
+                    <li>Slow Internet</li>
+                </a>
+            </ul>
+        </div>
+    </div>
+    <hr>
+
+
+    <div id="gallery">
```

Comment:
Really like what you did with the images in the gallery. Way to use CSS to make the "pop".